make/hotspot/lib/CompileJvm.gmk

```diff
@@ -146,6 +145,16 @@ $(call FillFindCache, $(JVM_SRC_DIRS))
 # operator new.
 LIBJVM_STATIC_EXCLUDE_OBJS := operator_new.o
 
+ifeq ($(call isTargetOs, windows), true)
+  WIN_EXPORT_FILE := $(JVM_OUTPUTDIR)/win-exports.txt
```

Comment:
Why not .def?

Reply:
I try to stick to established file suffixes if possible. I have never heard of .def before, but maybe that is a standard name for these kinds of files on Windows? If so, I will change it.